x-pack/plugin/sql/qa/server/src/main/resources/filter.sql-spec

```diff
@@ -22,6 +22,10 @@ whereFieldAndComparison
 // tag::whereFieldAndComparison
 SELECT last_name l FROM "test_emp" WHERE emp_no > 10000 AND emp_no < 10005 ORDER BY emp_no LIMIT 5;
 // end::whereFieldAndComparison
+whereGreaterThanAndNotEqualityOnDifferentFields
+// tag::whereGreaterThanAndNotEqualityOnDifferentFields
+SELECT last_name l FROM "test_emp" WHERE salary >= 50000 AND emp_no != 10002 ORDER BY emp_no LIMIT 5;
+// end::whereGreaterThanAndNotEqualityOnDifferentFields
```

Comment:
tag is used for documentation purposes. Unless you consume it into the docs, there's no point to add it.